container_test/handle_service_container_test.py

```diff
@@ -0,0 +1,132 @@
+import os
+import pytest
+import requests
+import time
+
+from typing import Optional
+from lib.AbstractHandle.AbstractHandleClient import AbstractHandle
+
+""" handle_service_container_test.py
+
+Very simple tests to ensure that local handle service server is functioning correctly.
+Requires the python libraries `pytest` and `requests` to be installed.
+
+Assumes that the handle_service2 is running locally on ports 8080.
+
+Use the wrapper shell script, `run_tests.sh`, to create the necessary set up and run the tests:
+
+sh scripts/run_tests.sh
+
+"""
+
+# setup TEST_TOKEN as env or provide a TOKEN_FILE_PATH
+# export TEST_TOKEN="your-kb-ci-token"
+TOKEN_FILE_PATH = None
+
+HANDLE_SERVICE_VERSION = "1.0.7"
+FILE_NAME = "mylittlefile"
+
+HS_URL = "http://localhost:8080"
+BLOB_URL = "https://ci.kbase.us/services/shock-api"
```

Comment:
If you want to make this truly standalone you could add auth and the blobstore tot the docker compose, similar to how the workspace does it with auth. Then you don't need a token. Maybe not enough ROI though.

Reply:
I think this is good for now. If you think is necessary, I can address this after I get myself unblocked from https://github.com/jgi-kbase/IDMappingService/pull/140

Reply:
I'm ambivalent